### DISCUSSION TITLE
Hyperlinks Policy

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#43] [Added]
### Type name

~Linking Policy~
Hyperlinks Policy https://github.com/OpenTermsArchive/terms-types/discussions/43#discussioncomment-10198751

### Alternative type names

~No response~
Links Policy, Linking Policy, External Links Policy, Outgoing Links Policy https://github.com/OpenTermsArchive/terms-types/discussions/43#discussioncomment-10198751

### Writer

service provider

### Audience

~publisher~
for outgoing hyperlinks, readers and contributors; for incoming hyperlinks, third-party authors who link to the service content https://github.com/OpenTermsArchive/terms-types/discussions/43#discussioncomment-10198751

### Object

~managing website hyperlinks~
quality of third-party targets of outgoing hyperlinks provided in content, and requirements for incoming hyperlinks in third-party content https://github.com/OpenTermsArchive/terms-types/discussions/43#discussioncomment-10198751

### Examples

| Example | Service Name | URL | Title | Jurisdiction |
|:----|:----|:----|:------|:-------------|
| 1 | City of Reading | [URL 🔗](https://www.readingpa.gov/linking-policy) | Linking Policy | - |
| 2 | Global Virtual Opportunities | [URL 🔗](https://gogvo.com/legal/external-links-policy.php) | EXTERNAL LINKS POLICY | - |
| 3 | Mobility Works | [URL 🔗](https://www.mobilityworks.com/external-linking-policy/) | External Linking Policy | - |
| 4 | NHS | [URL 🔗](https://www.nhs.uk/our-policies/linking-from-the-nhs-website/) | Linking from the NHS website | - |
| 5 | St. Jude Children's Research Hospital | [URL 🔗](https://www.stjude.org/legal/linking-policy.html) | Linking Policy | - |
| 6 | Japan Times | [URL 🔗](https://www.japantimes.co.jp/about-us/link-policy/) | External links/excerpts | - |
| 7 | tubeviewranker | [URL 🔗](https://tubeviewranker.com/assets/files/externallinkspolicy.html) | External Links | - |
| 8 | USA.gov | [URL 🔗](https://www.usa.gov/linking-policy) | Linking policy | - |

**Comment by MattiSG** [comment]
The provided examples cover either policies for outgoing links (how does the service choose / allow links to third parties) or incoming links (how the service expects links to its content to be made):

- 1: outgoing
- 2: outgoing and incoming
- 3: incoming
- 4: outgoing
- 5: incoming
- 6: incoming (+ quoting articles)
- 7: outgoing
- 8: outgoing and incoming

So a total of:

- outgoing: 3
- incoming: 3
- both: 2

When looking at the details, there is even a difference for the outgoing links provided by content providers, which are more published under a transparency / commitment angle; and outgoing links policies for user-generated content, which are more under a moderation angle.

Here are the three types that this perspective would lead to, not taking into account the practicality of their usage compared to a single type:

## Outgoing Links Policy

- writer: authoritative content provider
- audience: end user
- object: quality of third-party targets of hyperlinks provided in content

## Contributed Outgoing Links Policy

- writer: user generated content service provider
- audience: contributors
- object: quality of third-party targets of hyperlinks provided in content

## Incoming Links Policy

- writer: service provider
- audience: third-party authors who link to the service content
- object: context and way in which links are made

**Reply by MattiSG** [reply]
Thank you for this detailed decomposition! While relevant to understand the type, this level of detail would make choosing a type rather cumbersome, especially considering that several examples combine several subtypes.

I thus suggest to combine the triptychs into a single type:

## Name

Hyperlinks Policy

## Alternative names

Links Policy
Linking Policy
External Links Policy
Outgoing Links Policy

## Audience

for outgoing hyperlinks, readers and contributors; for incoming hyperlinks, third-party authors who link to the service content

## Object

quality of third-party targets of outgoing hyperlinks provided in content, and requirements for incoming hyperlinks in third-party content

**Reply by clementbiron** [reply]
I agree with your suggestions and have updated accordingly.
